modules/dashboard.xml

```diff
@@ -27,7 +27,7 @@
     <class>\Xibo\Widget\DashboardProvider</class>
     <dataCacheKey>%widgetId%_%displayId%</dataCacheKey>
     <type>dashboard</type>
-    <dataType></dataType>
+    <dataType>xibo-dashboard-service</dataType>
```

Comment:
Previously we fixed an issue with the dashboard module not getting updated tokens by adding `$dataModule->isWidgetProviderAvailable()` to where we output required files for data. However, that breaks all other modules with a provider that do not have data (we get a required file node which can never be fulfilled).

The correct fix for dashboards is to add a data type.